examples/README.md

```diff
@@ -95,6 +95,20 @@ Coming soon!
 | [**`translation`**](https://github.com/huggingface/transformers/tree/master/examples/seq2seq)                       | WMT             | ✅  | - | - | -
 
 
+
+## Resuming training
+
+Trainer supports resuming from a checkpoint via `trainer.train(resume_from_checkpoint)`.
+
+1. If `resume_from_checkpoint` is `True` it will look for the last checkpoint in the value of `output_dir` passed via `TrainingArguments`.
+2. If `resume_from_checkpoint` is a path to a specific checkpoint it will use that saved checkpoint folder to resume the training from.
```

Comment:
I don't think this part is useful here, it can go in the Trainer page documentation if this one is unclear. I would focus on the way to do it with the checkpoints only.

Reply:
I thought it'd be helpful for someone extending an example or turning it into notebook so that they can see a parallel and will know how to quickly switch to do the same in the trainer. These are examples after all.

But if you feel it's redundant I will remove it.

Reply:
We can put it in a second part then, but the first information should be how to do it in the scripts.

Reply:
excellent. Please have a look at the re-shuffled version

Reply:
Good to merge, thanks for adjusting!